oaknational/Oak-Web-Application
# [CUR-282] Previous/following unit description in unit modal

## Description

Music year: 1970

- Added future and prior unit descriptions and titles to unit modals in the curriculum page

### Acceptance criteria
- [x]  Prior unit description
    - [x]  Collapsable accordion section
    - [x]  When user clicks section:
        - [x]  Arrow changes state
        - [x]  Prior unit description appears
    - [x]  When user clicks box again
        - [x]  Section closes
- [x]  Future unit description
    - [x]  Collapsable accordion section
    - [x]  When user clicks section:
        - [x]  Arrow changes state
        - [x]  Prior unit description appears
    - [x]  When user clicks box again
        - [x]  Section closes

## How to test

1. Go to https://deploy-preview-2094--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/units
2. Click on any unit to expand the unit modal. 
3. You should see the prior and future unit descriptions
4. There should be no prior unit description for the first uni in a programme
## Screenshots

How it used to look (delete if n/a):
<img width="632" alt="Screenshot 2023-11-14 at 16 35 24" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/7ac2012b-c583-40d6-99a4-d0292ddd8e26">


How it should now look:
<img width="612" alt="Screenshot 2023-11-14 at 16 34 38" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/15d33e0b-0818-49dd-b62f-0c3825fe31c3">
<img width="666" alt="Screenshot 2023-11-14 at 16 34 58" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/8a0fc1ee-916c-4b25-9de4-732d356e9d25">

## Checklist
